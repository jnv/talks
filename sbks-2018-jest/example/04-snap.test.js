test('snapshot example', () => {
  const result = { some: 'object', other: 'stuff' }
  expect(result).toMatchSnapshot()
})