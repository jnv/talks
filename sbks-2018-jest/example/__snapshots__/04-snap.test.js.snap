// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot example 1`] = `
Object {
  "other": "stuff",
  "some": "object",
}
`;
